### PR TITLE
fix: `instanceof MedusaError` does not work

### PR DESCRIPTION
**Pull request by McTom234**
The same as #9094, but for v1. Our project won't be moving to v2 soon probably, and we want the feature of course :smile:

### CODE REVIEW

File: packages/utils/src/common/errors.ts [hunk 1]
```diff
@@ -26,6 +26,9 @@ export const MedusaErrorCodes = {
  * @extends Error
  */
 export class MedusaError extends Error {
+  // Symbol is not serializable
```
Comment (riqwan) [inline packages/utils/src/common/errors.ts]:
```suggestion
```

**Review by riqwan — APPROVED** [review]

**Comment by McTom234** [comment]
Do we need a changeset here?

**Comment by riqwan** [comment]
> Do we need a changeset here?

yes, for v1 we do

File: .changeset/tender-oranges-clean.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@medusajs/utils": minor
```
Comment (riqwan) [inline .changeset/tender-oranges-clean.md]:
```suggestion
"@medusajs/utils": patch
```

File: .changeset/tender-oranges-clean.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+add an identifier for MedusaError
```
Comment (riqwan) [inline .changeset/tender-oranges-clean.md]:
```suggestion
feat(utils): adds an identifier for MedusaError
```

**Comment by McTom234** [comment]
Okay, cool! `automerge` means it's merged automatically?

**Pull request merged**